Gemfile

```diff
@@ -1,33 +1,35 @@
-source "https://rubygems.org"
+source 'https://rubygems.org'
 git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
-ruby "3.1.2"
+ruby '3.1.2'
 
-gem "figaro", '~> 1.2.0'
+gem 'figaro', '~> 1.2.0'
+
+gem 'rubocop', '>= 1.0', '< 2.0'
 
 # Bundle edge Rails instead: gem "rails", github: "rails/rails", branch: "main"
-gem "rails", "~> 7.0.3"
+gem 'rails', '~> 7.0.3'
```

Comment:
- [ ] Kindly make sure to use the required gems: 
1. webpack
2. react-rails

Reply:
Hi @sja-thedude , Starting Rails 7, webpacker is no longer supported, you can check their official website [here](https://github.com/rails/webpacker)
We read about this a lot and there are mainly 2 ways to use react with rails 7 -
1. Using import-map
2. Using esbuild

Import-map doesn't let us use jsx syntax so I opted for esbuild. So this project is build with esbuild instead of webpacker and react-rails gem